HSTracker/UIs/DeckManager/NewDeck.swift

```diff
@@ -89,12 +89,18 @@ class NewDeck: NSWindowController {
                                             if let deck = deck {
                                                 self._addDeck(deck)
                                             } else {
-                                                // TODO show error
+                                                // show error
+                                                let alertDialog: NSAlert = NSAlert()
+                                                alertDialog.messageText = "Failed to import deck from \n" + self.urlDeck.stringValue
```

Comment:
Could you change this message to a NSLocalizedString call and add the message in all Localizable.strings `Failed to import deck from \n%@"` ?